src/component/handlers/edit/editOnBeforeInputAndroid.js

```diff
@@ -147,21 +154,7 @@ function editOnBeforeInputAndroid(editor: DraftEditor, e: InputEvent): void {
     }
 
     case 'insertFromPaste': {
-      // TODO: pastes will always be plaintext until we handle the dataTransfer object
-      const pasteChars = e.dataTransfer.getData('text/plain');
-
-      e.preventDefault();
-      editor.update(
-        replaceText(
-          editorStateWithCorrectSelection,
-          pasteChars,
-          editorStateWithCorrectSelection.getCurrentInlineStyle(),
-          getEntityKeyForSelection(
-            editorStateWithCorrectSelection.getCurrentContent(),
-            editorStateWithCorrectSelection.getSelection(),
-          ),
-        ),
-      );
+      // Allow insert `insertFromPaste` to pass through.
```

Comment:
Might want to point readers towards the paste handler you added

Reply:
Good point. Updated!